lib/web_ui/lib/src/engine/html/platform_view.dart

```diff
@@ -75,18 +75,28 @@ class PersistedPlatformView extends PersistedLeafSurface {
     }
   }
 
+  // Platform Views can only be updated if their viewId matches.
+  @override
+  bool canUpdateAsMatch(PersistedSurface oldSurface) {
+    if (super.canUpdateAsMatch(oldSurface)) {
+      if (oldSurface is PersistedPlatformView) {
```

Comment:
`super.canUpdateAsMatch` already checks the type, so here we should just cast (which I think will be optimized away by dart2js).

Reply:
I thought `is` checks didn't have much cost anymore. I'll remove this and call `(oldSurface as PersistedPlatformView).viewId`...

Reply:
Done